packages/table/src/index.ts

```
@@ -0,0 +1,799 @@
+            if (!line.trim()) {
+                result.push(line);
+                continue;
+            }
+
+            // Extract all color and link sequences
+            const segments: { color?: string; text: string; url?: string }[] = [];
+            let currentPos = 0;
+            let currentColor = "";
+
+            // Regular expression to match both color codes and hyperlinks
+            const pattern = /\u001B\[(\d+)m|\u001B\]8;;([^\u0007]*)\u0007([^\u0007]*)\u001B\]8;;\u0007/g;
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)